app/views/placeholder/landing.html.erb

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
+    <style>
+    html, body {
+      background-image: url('<%= asset_path('prisms.png') %>'); 
+      font-family: 'Roboto Slab';
+      height:100%; /* important to vertically align the container */
+      margin:0;
+      padding:0;
+    }
+    .vertical-center {
+      min-height: 100%;  /* Fallback for browsers do NOT support vh unit */
+      min-height: 100vh; /* These two lines are counted as one 🙂       */
+      display: flex;
+      align-items: center;
+    }
+    </style>
+    <title>Hello, world!</title>
+  </head>
+  <body>
+  
+  
+
+
+<div class="vertical-center" >
+  <div class="container rounded" style="background-color: #FFFFFF; max-width:1000px" >
+    <div class="row">
+      <div class="col-sm-12" style="text-align:center">
+          <%=image_tag("LEAD logo.png", :align=> "middle", :style => "width:300px")%>
+          <h1>Communities@Cal</h1>
+          <p>Please login through CalNet to continue.</p> 
+          <%= form_tag login_path, :method => 'post' do %>
+            <input type="submit" value="Placeholder Login" class="btn-primary"/>
+          <% end %>
+          <br>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>

```

Comment:
We should be taking care of adding jQuery and Bootstrap through the Gemfile instead
```
gem 'bootstrap'
gem 'jquery-rails'
```